examples/fscloud/main.tf

```diff
@@ -114,6 +132,16 @@ module "cos_fscloud" {
           name  = "networkZoneId"
           value = module.cbr_zone.zone_id
       }]
+      }, {
+      attributes = [
+        {
+          "name" : "endpointType",
+          "value" : "public"
```

Comment:
not sure we want public here? I believe that would enforce a rule that says schematics IP range can only talk with COS over public endpoint